core/pkg/resourcehandler/k8sresources.go

```diff
@@ -62,11 +62,6 @@ func (k8sHandler *K8sResourceHandler) GetResources(ctx context.Context, sessionO
 		return nil, nil, nil, nil, err
 	}
 
-	// we don't scan resources which have a parent
```

Comment:
this is already being checked on the `findScanObjectResource` function

Reply:
@amirmalka Can you please verify this?